server/discord.py

```diff
@@ -1,13 +1,12 @@
-from datetime import datetime, timedelta
 from time import sleep
-from typing import Generator, List, Tuple
+from typing import Generator, List
 
-from selenium.common.exceptions import StaleElementReferenceException, NoSuchElementException
+from selenium.common.exceptions import StaleElementReferenceException
 from selenium.webdriver.chrome.webdriver import WebDriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.remote.webelement import WebElement
 
-from consts import DISCORD_LOGIN_URL, SERVER_CLASS_NAME
+from server.consts import *
```

Comment:
Not sure about this line
I could write it all out explicitly, but also we would definitely want to use all the available exceptions.
What do you think?